_posts/aws/2019-06-03-AWS-api-gateway-integration.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2019-05-29
+title: AWS API Gateway Integration
+categories:
+   - AWS
+description: How to integrate AWS API Gateway to expose Spinnaker webhook
+type: Document
+---
+
+## Background
+
+There are a handful of options to manage the accessing to Spinnaker by third-party systems, without Spinnaker being exposed externally, this document will show you how to expose Spinnaker's resources thorugh [AWS API Gateway](https://aws.amazon.com/api-gateway/) which will help us to monitor and restrict access to our expose resources.
```

Comment:
Grammar:
`to manage access to Spinnaker`
`which will help us monitor and restrict access to our exposed resources.`